src/components/forms/SceneSelect.tsx

```diff
@@ -61,10 +61,11 @@ export const SceneSelect: FC<SceneSelectProps> = ({
   value,
   onChange,
   optional,
-  optionalLabel,
+  optionalLabels,
   ...selectProps
 }) => {
   const scenes = useAppSelector((state) => sceneSelectors.selectAll(state));
+  const optionalScenes = (optionalLabels || []).map((optionalLabel, i) => ({ value: i.toString(), label: optionalLabel }) ) as SceneOption[];
```

Comment:
When there was only one non-scene option, a `SceneOption` was created with a value of "". To facilitate more non-scene options, I use their index as the value. It's pretty hacky, but it's simple, easy to work with, and required minimal changes.